src/api/v1/users/register/__tests__/visitors.test.integration.ts

```diff
@@ -0,0 +1,136 @@
+import * as Hapi from 'hapi';
```

Comment:
The diff thinks this is a new file, but I'm assuming it's just been moved and renamed. Did you make any changes to the file?

Reply:
Think its a product of my workflow. I initially added `users/register/volunteers` in the same file, then decided to separate for clarity.

Reply:
OK, but have you actually changed any of the code in this file?

Reply:
OK, but have you actually changed any of the code in this file? I just want to know whether I need to pay attention to this or not.

Reply:
Not to my recollection (apart from the endpoint name)